java-util/src/main/java/io/druid/java/util/common/parsers/ParserUtils.java

```diff
@@ -94,4 +94,25 @@ public static String stripQuotes(String input)
     }
     return input;
   }
+
+  public static String replace(String text, String searchString, String  replacement)
```

Comment:
Would you elaborate more why this method is needed? There are ```String.replace()``` or ```String.replaceAll()```, but they aren't enough?